pkgs/unified_analytics/lib/src/survey_handler.dart

```diff
@@ -134,10 +134,12 @@ class Survey {
         samplingRate = json['samplingRate'] is String
             ? double.parse(json['samplingRate'] as String)
             : json['samplingRate'] as double,
-        excludeDashToolList =
-            (json['excludeDashTools'] as List<dynamic>).map((e) {
-          return DashTool.getDashToolByLabel(e as String);
-        }).toList(),
+        excludeDashToolList = (json['excludeDashTools'] as List<dynamic>)
+            .map((e) {
+              return DashTool.fromLabel(e as String);
+            })
```

Comment:
super nit, this can be a one liner: `.map((e) => DashTool.fromLabel(e as String))`